### DISCUSSION TITLE
Reopen #2758

### DISCUSSION THREAD

**Discussion by twsl** [python-telegram-bot/python-telegram-bot#2884] [Feature Suggestions]
Is there any chance to reopen https://github.com/python-telegram-bot/python-telegram-bot/issues/2758 ?
I'm having the same issue and having a pinned library makes things annoying. The suggest pin `cachetools>=4.2.2,<5` seems like a reasonable solution to the problem.

**Comment by Bibo-Joshi** [comment]
#2758 is in fact open. It will be addressed in v14.

**Reply by twsl** [reply]
I probably should have said unlocked. There is no way to interact with the issue at all.

**Reply by Bibo-Joshi** [reply]
I see. The thread was locked because the foregoing discussion in #2757 had gone off trail. I can unlock the thread. If you simply want to express support for changing the pinning policy, please use the emoji reactions (👍 ) instead of commenting on the thread. Note that the thread will be locked automatically again after 7 days of inactivity on the thread.
